### PR TITLE
IAV-2349 Missing translations

### PR DESCRIPTION
**Pull request by juhog**
https://jira.druid.fi/browse/IAV-2450

DoD, add these translations:
- Avoimet lähetteet listan "Laboratoriotutkimus (1234)"
- "Ei asiantuntijoita"
- "Ei varauksia"
- "Ei tutkimuksia"
- "Sinulla ei ole laboratoriotuloksia"

### CODE REVIEW

File: src/assets/i18n/sv/common.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "Yes": "Ja",
   "No": "Nej",
+  "Error!": "Fel!",
   "days": "d",
-  "Cancel": "Avboka",
   "Save": "Spara",
+  "____Saving": "",
```
Comment (kallevu) [inline src/assets/i18n/sv/common.json]: Why this is needed? And only in Swedish?

File: src/assets/i18n/fi/laboratory.json [hunk 2]
```diff
@@ -4,5 +4,6 @@
   "Reference Range": "Viitearvo",
   "Unit": "Yksikkö",
   "Your laboratory test results will be posted on your account when they have been completed. The doctor responsible for your care will discuss the results with you as arranged. Laboratory staff are not authorised to discuss your test results. You can view your treatment plan also in {{link}}.": "Näet laboratoriovastaukset niiden valmistuttua. Hoitava lääkärisi käy tulokset sovitusti läpi kanssasi. Laboratorion henkilökunnalla ei ole valtuuksia tulkita vastauksia. Voit katsoa oman hoitosuunnitelmasi myös {{linkFi}}.",
-  "Select year": "Valitse vuosi"
+  "Select year": "Valitse vuosi",
+  "No laboratory results were found for you": "Sinulla ei ole laboratoriotuloksia"
```
Comment (kallevu) [inline src/assets/i18n/fi/laboratory.json]: Should we make empty values consistent by always using passive or adding "for you" for other empty values?

File: src/assets/i18n/fi/common.json [hunk 3]
```diff
@@ -4,26 +4,27 @@
...
-  "You can no longer cancel this appointment online.": "Aikaa ei voi enää perua netissä",
-  "My reservations": "Omat varaukset",
-  "Summary": "Yhteenveto",
   "Diagnoses": "Diagnoosit",
   "Diagnosis": "Diagnoosi",
+  "Pending referrals": "Avoimet lähetteet",
+  "No specialists": "Ei asiantuntijoita",
+  "No appointments": "Ei varauksia",
+  "No tests": "Ei tutkimuksia",
+  "No referrals": "Ei avoimia lähetteitä",
```
Comment (kallevu) [inline src/assets/i18n/fi/common.json]: We should move `No diagnoses were found for you` string from diagnoses.json to common.json.

File: src/assets/i18n/sv/common.json (hunk 1 above)
Comment (juhog) [inline src/assets/i18n/sv/common.json]: (Notes from our hangouts call)

Marker for a missing translation (to keep translation files organized).

**Pull request merged**